ferrocene/doc/sphinx-shared-resources/exts/ferrocene_domain_cli/domain.py

```diff
@@ -149,3 +171,9 @@ def warn(message, location):
 
 def setup(app):
     app.add_domain(CliDomain)
+
+
+@dataclass
+class ProgramStorage:
+    program_name: str = "PLACEHOLDER"
+    no_traceability_matrix: bool = False
```

Comment:
We don't have to provide default values, just this would work:

```suggestion
    program_name: str
    no_traceability_matrix: bool
```

Reply:
👍🏾